buttons/src/main/java/com/decathlon/vitamin/compose/buttons/VitaminButtons.kt

```diff
@@ -341,6 +341,7 @@ private fun VitaminButtonImpl(
         ) {
             if (iconSide == IconSide.LEFT) iconButton()
             Text(
+                modifier = Modifier.weight(1f, fill = false),
```

Comment:
Because we have already snapshot tests for buttons, that could be great to add this in our current tests no? :)